BLSQ/iaso
# Polio-1392 budget timeline layout

The divider had disappeared, icon buttons were decentered in mobile layout

Related JIRA tickets :POLIO-1392

## Self proofreading checklist


## Changes

Explain the changes that were made. The idea is not to list exhaustively all the changes made (GitHub already provides a full diff), but to help the reviewers better understand:
- which specific file changes go together, e.g: when creating a table in the front-end, there usually is a config file that goes with it
- the reasoning behind some changes, e.g: deleted files because they are now redundant
- the behaviour to expect, e.g: tooltip has purple background color because the client likes it so, changed a key in the API response to be consistent with other endpoints

## How to test

Witha polio Db witha budget config (ie: a budget workflow json, and [teams configured](https://wiki.bluesquare.org/en/Dev-team/product-management/iaso/Polio/polio_budget_workflow)):

-  Go to polio budget page
-  switch to mobile view: the icon button in the card should be centered
-  switch back to web view
-  Try to create a step: the `+` icon button to add a link should be aligned with the text
-  Repeat this step in mobile
-  Repeat both steps above with the `override` button
-   Have a look at the timeline widget: there should be a vertical line (divider) linking the "title" (eg: RRT finances") with the list of steps

## Print screen / video
![Screenshot 2024-01-09 at 10 41 55](https://github.com/BLSQ/iaso/assets/38907762/4b162eff-8957-44d0-b4b6-e3779f1b17ff)

![Screenshot 2024-01-09 at 10 53 25](https://github.com/BLSQ/iaso/assets/38907762/08b5f774-dafe-4a23-8504-73e5f5d49e40)
![Screenshot 2024-01-09 at 10 54 02](https://github.com/BLSQ/iaso/assets/38907762/66a29db7-d1e8-4eb1-ae44-dda122ea941e)

## Notes

merges into POLIO-1393_remove_link_to_procedure
